clang/lib/CIR/CodeGen/CIRGenCleanup.h

```diff
@@ -257,5 +258,53 @@ inline void EHScopeStack::popCatch() {
   deallocate(EHCatchScope::getSizeForNumHandlers(scope.getNumHandlers()));
 }
 
+/// The exceptions personality for a function.
+struct EHPersonality {
```

Comment:
Is is possible to abstract most of this (or if you tried) into a base class that could be shared between it and classic CG? Seems like 95% is the same but `::get` methods. It can come as follow up PR, but I think we should handle this sooner than later. (cc @andykaylor)

Reply:
Sure, I will do that 🫡